hazelcast/src/test/java/com/hazelcast/map/MapUnboundReturnValues_StringKeyTest.java

```diff
@@ -0,0 +1,27 @@
+package com.hazelcast.map;
+
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.annotation.SlowTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category(SlowTest.class)
+public class MapUnboundReturnValues_StringKeyTest extends MapUnboundReturnValuesTestSupport {
```

Comment:
BTW: `MapUnboundReturnValues_PreCheckTest.testMapKeySet_SmallLimit()` takes 16 to 17 seconds on my local machine. I think that is a `SlowTest`.